Jellyfin.Data/Enums/MediaSegmentType.cs

```diff
@@ -0,0 +1,33 @@
+namespace Jellyfin.Data.Enums
+{
+    /// <summary>
+    /// Enum MediaSegmentType.
+    /// </summary>
+    public enum MediaSegmentType
```

Comment:
A general purpose `Annotation` type would be useful to mark noteworthy parts with comments or alike.

Reply:
Is this is a hint for me or a side note?

Reply:
a little bit of both. If there are no vetos i would _recommend_ adding this type too as it would also very easily add another feature request. But then some sort of comment field in `MediaSegment.Comment` would also be very useful to fill it with custom content.